spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/system/DiskSpaceHealthContributorAutoConfigurationTests.java

```diff
@@ -71,4 +99,13 @@ void runWhenDisabledShouldNotCreateIndicator() {
 				.run((context) -> assertThat(context).doesNotHaveBean(DiskSpaceHealthIndicator.class));
 	}
 
+	@SafeVarargs
```

Comment:
Yuk....

Alternative is to use `List<Map.Entry<K, V> entries>` which then leads to the tests having to use something like `Arrays.asList(...)`.  I went w/ convenience of callers namely because this is test code.